geopandas/base.py

```diff
@@ -538,6 +538,51 @@ def convex_hull(self):
         """
         return _delegate_property("convex_hull", self)
 
+    def delaunay_triangles(self, tolerance=0.0, only_edges=False):
+        """Returns a ``GeoSeries`` of shapely ``GeometryCollection`` objects
```

Comment:
Can we keep the first line short and leave the details for other lines? The first one shows up in the API (see [here](https://geopandas--2907.org.readthedocs.build/en/2907/docs/reference/geoseries.html)) and is better to keep it concise.